selfdrive/car/nissan/carstate.py

```diff
@@ -0,0 +1,143 @@
+import copy
+from common.kalman.simple_kalman import KF1D
+from selfdrive.config import Conversions as CV
+from opendbc.can.parser import CANParser
+from selfdrive.car.nissan.values import DBC
+
+def get_powertrain_can_parser(CP):
```

Comment:
**Recently made improvement in car abstraction (see other cars)**
Make this a static method of CarState and rename it `get_can_parser`